crates/era-cheatcodes/src/cheatcodes.rs

```diff
@@ -640,6 +644,54 @@ impl<S: DatabaseExt + Send, H: HistoryMode> VmTracer<EraDb<S>, H> for CheatcodeT
                         is_service: false,
                     });
                 }
+                FinishCycleOneTimeActions::MakePersistentAccount { account } => {
+                    let handle: &ForkStorage<RevmDatabaseForEra<S>> =
+                        &storage.borrow_mut().storage_handle;
+                    let fork_storage = handle.inner.write().unwrap();
+                    let era_db = fork_storage.fork.as_ref().unwrap().fork_source.clone();
+
+                    let mut db = era_db.db.lock().unwrap();
+                    db.add_persistent_account(revm::primitives::Address::from(
+                        account.to_fixed_bytes(),
+                    ));
+                }
+                FinishCycleOneTimeActions::MakePersistentAccounts { accounts } => {
+                    let handle: &ForkStorage<RevmDatabaseForEra<S>> =
+                        &storage.borrow_mut().storage_handle;
+                    let fork_storage = handle.inner.write().unwrap();
+                    let era_db = fork_storage.fork.as_ref().unwrap().fork_source.clone();
+
+                    let mut db = era_db.db.lock().unwrap();
+                    db.extend_persistent_accounts(
+                        accounts
+                            .into_iter()
+                            .map(|a: H160| revm::primitives::Address::from(a.to_fixed_bytes()))
+                            .collect::<Vec<revm::primitives::Address>>(),
+                    );
+                }
+                FinishCycleOneTimeActions::RevokePersistentAccount { account } => {
+                    let handle: &ForkStorage<RevmDatabaseForEra<S>> =
```

Comment:
Are these type annotations required here?

Reply:
Not really, I'll remove it from the one time actions.